manifests/server/options.pp

```diff
@@ -22,6 +22,7 @@
   $check_names_slave = undef,
   $check_names_response = undef,
   $allow_query = [],
+  $no_dns_sec = false,
```

Comment:
What if you named this just "dnssec_validation" and made it default to true? I think that would be easier to read than a double negative.